Assets/animController.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections;

```

Comment:
Just realized what this file was for, use the `KeyboardTester.cs` for this, it is a testing script so doesn't need to end up in the main repo